go.mod

```diff
@@ -34,9 +34,9 @@ require (
 	github.com/sigstore/fulcio v1.7.1
 	github.com/sigstore/protobuf-specs v0.4.3
 	github.com/sigstore/rekor v1.3.10
-	github.com/sigstore/rekor-tiles v0.1.6
+	github.com/sigstore/rekor-tiles v0.1.7-0.20250624231741-98cd4a77300f
 	github.com/sigstore/sigstore v1.9.5
-	github.com/sigstore/sigstore-go v1.0.0
+	github.com/sigstore/sigstore-go v1.0.1-0.20250710071215-2d48428552e7
```

Comment:
I'm using the latest sigstore-go commit currently since I need SigningConfig v0.2 support.